elyra/pipeline/resources/palette.json

```diff
@@ -5,7 +5,7 @@
       "label": "Notebook",
       "image": "",
       "id": "notebooks",
-      "description": "Notebook",
+      "description": "Notebook file",
```

Comment:
If this is the component description in the palette, it should probably read "Run Jupyter notebook" because a node implements a task, not a file of that type.

Reply:
I opened #1859 to address this since I saw your comment on the pipeline editor PR. It can definitely just as easily be changed here, I just wasn't sure if it would have an unintended effects on the front end.

Reply:
So, it is an issue in Master today, right? so maybe leave them as is in Master and use #1859 to make changes on places as needed?